src/components/Controls.vue

```diff
@@ -253,6 +259,7 @@ export default {
 		},
 		toggleCompactMode() {
 			this.$store.dispatch('toggleCompactMode')
+			localStorage.setItem('nc_deck_compact_active', this.compactMode)
```

Comment:
Please move the localStorage handling to the store where toggleCompactMode is actually taking place.

The getItem can then also be fetched as initial value for the store data.

Reply:
How to add an created function in the main.js store to fetch the store data?

Reply:
You can't and don't need one. You can just set it in the initial state of the store.

Reply:
How to do this? This is not working (main.js)
state: {
		showArchived: false,
		navShown: true,
		**compactMode: localStorage.getItem('deck.compactMode'),**

Reply:
You don't set it in your current pull request, once it is set that works like a charm here.

Reply:
Changed it, it's not working for me. 
On every page reload it's compact